build.sbt

```diff
@@ -553,7 +554,11 @@ lazy val httpClientZioBackend =
   httpClientBackendProject("zio")
     .settings(
       libraryDependencies ++=
-        Seq("dev.zio" %% "zio" % zioVersion)
+        Seq(
+          "dev.zio" %% "zio" % zioVersion,
+          "dev.zio" %% "zio-streams" % zioVersion,
+          "dev.zio" %% "zio-interop-reactivestreams" % "1.0.3.5-RC10"
```

Comment:
isn't the same used for `asyncHttpClientZioStreamsBackend`? if so, we should extract the version number to a constant